cpp/CMakeLists.txt

```diff
@@ -351,7 +376,7 @@ if(BUILD_CUML_CPP_LIBRARY)
       CUDA::cudart
       CUDA::cusparse
       $<$<BOOL:${NVTX}>:CUDA::nvToolsExt>
-      FAISS::FAISS
+      $<$<NOT:$<BOOL:${EXPERIMENTAL_FIL_ONLY}>>:FAISS::FAISS>
```

Comment:
I'm not immediately seeing where `EXPERIMENTAL_FIL_ONLY` is coming from?

Reply:
pffft totally forgot that initially I had a variable named like that... thanks for the catch!